### ISSUE TITLE
about the 'col_pred' key in the folder 'data' for the bert version

### ISSUE THREAD

**Issue by Twsschx** [microsoft/IRNet#42]
I have run the code with the version of bert. But wthat does the key 'col_pred' in the dev and train data mean during the "to_batch_seq" procedure and how to generate these values?
Because I couldn't find the data preprocess for this part.

**Comment by slamandar** [comment]
Hi, have you solved this problem? I met this problem as well and still do not solve it.

**Comment by AoZhang** [comment]
'col_pred' will only be used in inference mode. so you can just use IRNet released dev set instead of preprocessing by yourself.

**Comment by Twsschx** [comment]
Yes. Can you explain how to get the ‘col_pred’ result? Thanks a lot.

**Comment by marlesson** [comment]
+1

**Comment by papsipublic** [comment]
I would like to run my own question and to see the sql.
Is this column, "col_pred" important for new questions? 
If yes, how can we generate it? 

I would appreciate your answer very much.

**Comment by ygxw0909** [comment]
I also want to preprocess my own dataset, how can I get "col_pred"?
